src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -137,10 +137,16 @@ try
 {
     const auto drawingContext = static_cast<const DrawingContext*>(clientDrawingContext);
 
-    const DWRITE_FONT_WEIGHT weight = _fontRenderData->DefaultFontWeight();
+    DWRITE_FONT_WEIGHT weight = _fontRenderData->DefaultFontWeight();

```

Comment:
You can use `auto` here, but no need to change, as it's already been that way before your change.

Reply:
Thanks. But I actually want to make it abundantly clear what these types are. I think it helps people understanding the code & perform navigation to definition things.